docs/dev/api_concepts.md

```diff
@@ -896,11 +896,16 @@ result type ```R``` for the final result. E.g. for a histogram, ```V``` is a num
 
 {% top %}
 
-## EXPERIMENTAL: Reinterpreting a pre-partitioned data stream as keyed stream
+Experimental features
```

Comment:
Ah, what I meant is in a completely separate section of the doc, because this part of the doc is about general API concepts and not specific to the streaming API.